src/Sylius/Component/Grid/spec/Definition/FieldSpec.php

```diff
@@ -91,4 +91,12 @@ function it_can_have_options()
         $this->setOptions(['template' => 'SyliusUiBundle:Grid/Field:_status.html.twig']);
         $this->getOptions()->shouldReturn(['template' => 'SyliusUiBundle:Grid/Field:_status.html.twig']);
     }
+
+    function it_has_position_no_position_by_default()
```

Comment:
`it_has_no_position_by_default`. And there should be two methods: `no_position_by_default` and `mutable_position`